Source/Events.Streams/StreamPositionRange.cs

```diff
@@ -27,11 +27,11 @@ public StreamPositionRange(StreamPosition from, StreamPosition to)
         public StreamPosition From { get; }
 
         /// <summary>
-        /// Gets the, inclusive, end of the range.
+        /// Gets the length of the wanted range.
         /// </summary>
-        public StreamPosition To { get; }
+        public ulong Length { get; }
 
         /// <inheritdoc/>
-        public override string ToString() => $"({From}, {To})";
+        public override string ToString() => $"({From}, {Length})";
```

Comment:
The ToString could instead of being (from, length) be (from, to) where to = from + length. Then it would be the same as was before

Reply:
Does that make sense?